cmd/clusterctl/cmd/root.go

```diff
@@ -20,45 +20,98 @@ import (
 	"flag"
 	"fmt"
 	"os"
+	"strings"
 
+	"github.com/MakeNowJust/heredoc"
 	"github.com/spf13/cobra"
-	cliflag "k8s.io/component-base/cli/flag"
 	"k8s.io/klog"
 )
 
-const deprecationMsg string = "NOTICE: clusterctl has been deprecated in v1alpha2 and will be removed in a future version."
-
-var helpTemplate = fmt.Sprintf(`%s
-{{with (or .Long .Short)}}{{. | trimTrailingWhitespaces}}
-
-{{end}}{{if or .Runnable .HasSubCommands}}{{.UsageString}}{{end}}`, deprecationMsg)
+var cfgFile string
 
 var RootCmd = &cobra.Command{
 	Use:   "clusterctl",
-	Short: "cluster management",
-	Long:  `Simple kubernetes cluster management`,
-	RunE: func(cmd *cobra.Command, args []string) error {
-		return cmd.Help()
-	},
+	Short: "clusterctl controls a management cluster for Cluster API",
+	Long: LongDesc(`
```

Comment:
I would encourage the use of `dedendt.Dedent` here. Kubernetes already has a precedent for this library for this exact situation:

https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/config.go#L235

Reply:
If I may ask, what is the rationa to use dedent vs heredoc (which is used by kubectl)?

Reply:
Oh, I thought it would help by letting us remove the normalizer boilerplate slightly below, but if we still need the normalizer stuff then I don't care which library is used

Reply:
Ok, let start this way and eventually rework later